### ISSUE TITLE
[Bug]:  Banner Color Contrast not accesible

### ISSUE THREAD

**Issue by BeeSeeWhy** [sfbrigade/support-sfusd#385]
### What happened?

Element has insufficient color contrast

<img width="817" height="392" alt="Image" src="https://github.com/user-attachments/assets/f56ba52e-fe36-43aa-b05e-400157a3db70" />

### What should have happened instead?

Need to think about redesign of banner due to accessibility issues

### Version

supportsfschools.org (Production)

### What browser(s) are you seeing the problem on?

_all

### Which platform(s) are you seeing the problem on?

_all

**Comment by kdavis1924** [comment]
same issue as documented in #397
